soft-delete/packages.config

```diff
@@ -1,15 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.KeyVault.Core" version="2.0.4" targetFramework="net462" />
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net462" />
```

Comment:
What happened to `Azure.Storage.Blobs` ?

Reply:
Done.